.github/workflows/verify-bytecode.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Verify deployed contracts' bytecode
+
+on:
+  pull_request:
+    branches:
+      - "main"
+
+jobs:
+  assert-bytecode:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Setup node.js version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Cache node_modules
```

Comment:
is it really needed? you can use `cache` arg from actions/setup-node action as from another workflows

Reply:
Takeaway from `setup-node` action README:

> Note: The action does not cache node_modules